packages/integration-tests/src/relations/OneToManyLargeCollection.test.ts

```diff
@@ -33,8 +33,7 @@ describe("OneToManyLargeCollection", () => {
   });
 
   it("cannot be a load hint", async () => {
-    // ...actually this does work, b/c Tag doesn't have any other loadable collections,
-    // so NestedLoadHint becomes {}, and turns out anything extends {}...
+    // @ts-expect-error
     const hint: LoadHint<Tag> = { publishers: {} };
```

Comment:
This is actually supposed to be a compile error, and the reworking of `LoadHint` somehow "fixed" it to actually be a compile error. No idea why. :shrug:  :-)